tests/unit/api.spec.js

```diff
@@ -0,0 +1,161 @@
+import { standardizeTransactions } from '@/api'
+
+const TRANSACTIONS_RESULT = {
+  result: {
+    total: '24',
+    transactions: [
+      {
+        amount: '30000',
+        transaction: {
+          block: {
+            block_hash:
+              'f035340737b91c5d4b809f8780d9c7d236d05285cf5fde107048b764bf2d860f',
+            epoch: '5613152',
+          },
+          confirmed: true,
+          data: {
+            value_transfer: {
+              inputs: [
+                {
+                  address: 'twit1py55qev2etver5n8ehy9z6y0sxzthzef7ewr0h',
+                  value: '125000000000',
+                },
+              ],
+              outputs: [],
+            },
+          },
+          hash: '3d0162a7233c820dda9fc8b7d59909c9d8a733dde0a48642ed3f98948c2e1171',
+          miner_fee: '3000',
```

Comment:
If the transaction has 0 outputs, the miner fee will be equal to the input value:

```suggestion
          miner_fee: '125000000000',
```